packages/bot/src/constants/contract-stage.js

```diff
@@ -1,10 +1,10 @@
 import { translate } from '../utils/tools';
 
-export const CONTRACT_STAGES = {
-    not_running      : { index: 0 , text: translate('Bot is not running') },
-    starting         : { index: 1 , text: translate('Bot is starting') },
-    purchase_sent    : { index: 2 , text: translate('Attempting to Buy') },
-    purchase_recieved: { index: 3 , text: translate('Buy Succeeded') },
-    is_stopping      : { index: 4 , text: translate('Bot is stopping') },
-    contract_closed  : { index: 5 , text: translate('Contract Closed') },
-};
+export const contract_stages = Object.freeze({
+    NOT_RUNNING      : { index: 0 , text: translate('Bot is not running') },
+    STARTING         : { index: 1 , text: translate('Bot is starting') },
+    PURCHASE_SENT    : { index: 2 , text: translate('Attempting to Buy') },
+    PURCHASE_RECIEVED: { index: 3 , text: translate('Buy Succeeded') },
```

Comment:
This is a confusing typo used throughout the trade engine, can we fix it here at least. 😄 

```suggestion
    PURCHASE_RECEIVED: { index: 3 , text: translate('Buy Succeeded') },
```